core/src/avm2/globals/flash/text/StyleSheet.as

```diff
@@ -0,0 +1,39 @@
+package flash.text {
+    
+    import __ruffle__.stub_constructor;
+    import __ruffle__.stub_method;
+    import __ruffle__.stub_getter;
+    
+    public dynamic class StyleSheet {
+        public function StyleSheet() {
+            stub_constructor("flash.text", "StyleSheet");
```

Comment:
Does this need to be stubbed? Flash doesn't seem to suggest it actually *does* anything, so an empty constructor is a valid implementation